src/main/java/com/notably/logic/commands/NewCommand.java

```diff
@@ -0,0 +1,44 @@
+package com.notably.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import com.notably.commons.core.path.AbsolutePath;
+import com.notably.commons.core.path.RelativePath;
+import com.notably.commons.core.path.exceptions.InvalidPathException;
+import com.notably.logic.commands.exceptions.CommandException;
+import com.notably.model.Model;
+import com.notably.model.block.Block;
+
+/**
+ * Represent a command used to add a new block.
+ */
+public class NewCommand extends Command {
+    public static final String COMMAND_WORD = "new";
+    private final Block toAdd;
+    private RelativePath path;
+
+    public NewCommand(Block block, RelativePath path) {
+        requireNonNull(block);
+        this.toAdd = block;
+        this.path = path;
+    }
```

Comment:
Any reasons as to why it's `RelativePath` here, while it's `Path` in `DeleteCommand`, for example> I think we should keep the `Path` usage consistent with other command types, unless absolutely necessary.

Reply:
I figured that we can only create a `NewCommand` relative to the current directory. Thus i only allowed it to accept RelativePath. But as you've pointed out i think it should remain consistent with the other Command. So i've changed it to `Path` instead.